packages/stark-core/package.json

```diff
@@ -22,9 +22,33 @@
     "npm": ">=5.3.0"
   },
   "dependencies": {
+    "@angular/common": "5.2.6",
     "@angular/core": "5.2.6",
-    "rxjs": "5.5.6"
+    "@angular/platform-browser": "5.2.6",
+    "@angular/platform-browser-dynamic": "5.2.6",
+    "@nationalbankbelgium/stark-build": "file:../../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.0.0-alpha.0-a75807f.tgz",
```

Comment:
If it is not already the case, we should update the build.sh script for changing the dependency during the development